includes/CMB2_Base.php

```diff
@@ -379,6 +379,43 @@ public static function maybe_hook( $val, $hook_name, $hook_function ) {
 	}
 
 	/**
+	 * Arguments for custom wp_kses sanitization/validation calls.
+	 *
+	 * @since  2.2.4
+	 * @param  string $type The field type being escaped.
+	 * @return array        An array of wp_kses HTML parameters allowed.
+	 */
+	public function kses_args( $type ) {
+		$params = array();
+
+		switch ( $type ) {
+			case 'group_wrap_attributes':
+				$params = array(
+					'class' => array(),
+					'style' => array(),
+				);
+				break;
+			case 'remove_disabled':
+				$params = array(
+					'disabled' => array(),
+				);
+		}
+
+		return $params;
+	}
+
+	/**
+	 * Helper function that returns whether a field group is repeatable or not.
+	 *
+	 * @since  2.2.4
+	 * @param  object $field_group The field group.
+	 * @return boolean
+	 */
+	public function is_repeatable( $field_group ) {
```

Comment:
`$this->is_repeatable( $field_group )` is there for a couple reasons, and I'm happy to pull it out.

1. Scrutinizer was complaining about the repetition of `$field_group->args( 'repeatable' )` and it was done in an attempt to "leave the code better than you found it."
2. To Scrutinizer's point, it's done a whole bunch of times, and anything that's done more than once can be split into it's own function (admittedly, the function is just a single line that spits out the same output, so the value is questionable except...)
3. It's a couple less characters to type.

Ideally, the way I originally conceived it, it would be more global like `is_repeatable()` instead of being a class method that requires a parameter, but that gets hairy, too.

I like the `CMB2_Field::is( $param )` and being able to use it as `$field_group->is( 'repeatable' )` idea, though. It seems like an obvious evolution of the same idea.